apps/desktop-wallet/src/api/queries/addressQueries.ts

```diff
@@ -20,153 +20,101 @@ import { AddressHash, PAGINATION_PAGE_LIMIT, throttledClient } from '@alephium/s
 import { AddressTokenBalance } from '@alephium/web3/dist/src/api/api-explorer'
 import { queryOptions, skipToken } from '@tanstack/react-query'
 
-import { AddressLatestTransactionHashesProps } from '@/api/queries/transactionQueries'
-import queryClient from '@/api/queryClient'
+import { AddressLatestTransactionQueryProps } from '@/api/queries/transactionQueries'
 import { DisplayBalances, TokenDisplayBalances, TokenId } from '@/types/tokens'
 
-const ADDRESS_BALANCE_QUERY_KEYS = ['address', 'balance']
-
 export type AddressAlphBalancesQueryFnData = {
   addressHash: AddressHash
   balances: DisplayBalances
 }
 
-// Adding latestTxHash in queryKey ensures that we'll refetch when new txs arrive.
 // Adding networkId in queryKey ensures that switching the network we get different data.
 // TODO: Should we add explorerBackendUrl instead?
-export const addressAlphBalancesQuery = ({
-  addressHash,
-  networkId,
-  latestTxHash,
-  previousTxHash,
-  skip
-}: AddressLatestTransactionHashesProps) => {
-  const getQueryOptions = (latestTxHash: AddressLatestTransactionHashesProps['latestTxHash']) =>
-    queryOptions({
-      queryKey: [...ADDRESS_BALANCE_QUERY_KEYS, 'ALPH', { addressHash, latestTxHash, networkId }],
-      queryFn: !skip
-        ? async () => {
-            const balances = await throttledClient.explorer.addresses.getAddressesAddressBalance(addressHash)
-
-            return {
-              addressHash,
-              balances: {
-                totalBalance: BigInt(balances.balance),
-                lockedBalance: BigInt(balances.lockedBalance),
-                availableBalance: BigInt(balances.balance) - BigInt(balances.lockedBalance)
-              }
+export const addressAlphBalancesQuery = ({ addressHash, networkId, skip }: AddressLatestTransactionQueryProps) =>
+  queryOptions({
+    queryKey: ['address', addressHash, 'balance', 'ALPH', { networkId }],
```

Comment:
Yeah, kinda. That's what saw that the docs are also using. But it's a convention, we don't need to do it that way. I also thought of moving it as the very first argument so that when we which network we invalidate all queries of this network... But for now I just kept it as we had it.

Reply:
All good for me